loader/src/packfile_io.rs

```diff
@@ -62,6 +67,12 @@ impl Drop for PackfileMessageReaderFile {
     }
 }
 
+#[derive(PartialEq)]
+enum RuntimeType {
```

Comment:
We could expose this if you think it's worthwhile. Or we could remove it and just use #[cfg(not(target_arch = "wasm32"))] directly as needed

Reply:
Best to keep the public API minimal for now, until our internals stabilize a bit.